commit a6491f45dc1f7a32569aca9e16dbbc7959c8e5d9 (refs/changes/28/2028/1)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2017-06-01T17:18:13-07:00 (2 years, 4 months ago)
    
    MANTA-3291 minnow needs to use new moray, cueball
    MANTA-3292 minnow should set cueball pool "maximum" higher

diff --git a/package.json b/package.json
index 9ecd65a..119e45d 100644
--- a/package.json
+++ b/package.json
@@ -15,7 +15,7 @@
         "cmdutil": "1.1.0",
         "dashdash": "1.3.2",
         "libmanta": "git+ssh://git@github.com:joyent/node-libmanta.git#master",
-        "moray": "3.0.0",
+        "moray": "3.2.0",
         "node-uuid": "1.4.0",
         "once": "1.2.0",
         "statvfs": "2.0.0",
diff --git a/sapi_manifests/minnow/template b/sapi_manifests/minnow/template
index 763fd7f..acaf716 100644
--- a/sapi_manifests/minnow/template
+++ b/sapi_manifests/minnow/template
@@ -17,7 +17,7 @@
                         "srvDomain": "{{STORAGE_MORAY_SHARD}}",
                         "cueballOptions": {
                                 "target": 1,
-                                "maximum": 1,
+                                "maximum": 4,
                                 "resolvers": [ "nameservice.{{DOMAIN_NAME}}" ]
                         }
                 }
